### PR TITLE
[Debugger] bumped VsCodeDebugProtocol version to 16.3

### PR DESCRIPTION
**Pull request by jstedfast**

### CODE REVIEW

File: version-checks [hunk 1]
```diff
@@ -17,8 +17,8 @@ DEP[0]=md-addins
 DEP_NAME[0]=MDADDINS
 DEP_PATH[0]=${top_srcdir}/../md-addins
 DEP_MODULE[0]=git@github.com:xamarin/md-addins.git
-DEP_NEEDED_VERSION[0]=74cbf1295c7a35b62998824b070f38841183e7c7
-DEP_BRANCH_AND_REMOTE[0]="master origin/master"
+DEP_NEEDED_VERSION[0]=fdf439af0766800f01671b7163c7b1dce8079a38
+DEP_BRANCH_AND_REMOTE[0]="jstedfast-debugger-vscodedebugprotocol-bump-16.3.30820.2 origin/jstedfast-debugger-vscodedebugprotocol-bump-16.3.30820.2"
```
Comment (Therzok) [inline version-checks]: This is not needed if you have matching branch names.

Comment (jstedfast) [reply version-checks]: Oh, ok.

**Comment by sgmunn** [comment]
re-deploying tests to see if transient error.

**Comment by jstedfast** [comment]
@monojenkins rebase

**Comment by jstedfast** [comment]
binary compat error is unrelated

**Comment by sgmunn** [comment]
@ketttY can you run the debugger test cases with this build please

**Pull request closed without merging**